repl/src/main/scala/ammonite/repl/Storage.scala

```diff
@@ -22,6 +23,14 @@ trait Storage{
   val ivyCache: StableRef[IvyMap]
   def compileCacheSave(path: String, tag: String, data: CompileCache): Unit
   def compileCacheLoad(path: String, tag: String): Option[CompileCache]
+  def classFilesListSave(pkg: String,
+                         wrapper: String,
+                         dataList: List[(String, String)],
+                         imports: Imports,
+                         tag: String): Unit
+  def classFilesListLoad(pkg: String,
+                         wrapper: String,
+                         cacheTag: String): (Seq[CompileCache], Imports)
```

Comment:
We should not overload empty sequences to mean the cached value was not found; this should really return `Option[(Seq[CompileCache], Imports)]`